go/release-checklist.md

```diff
@@ -49,6 +49,7 @@ Fedora packaging:
 {%- if do_fast_track %}
  - [ ] [Submit a fast-track](https://github.com/coreos/fedora-coreos-config/actions/workflows/add-override.yml) for FCOS testing-devel
  - [ ] [Submit a fast-track](https://github.com/coreos/fedora-coreos-config/actions/workflows/add-override.yml) for FCOS next-devel if it is [open](https://github.com/coreos/fedora-coreos-pipeline/blob/main/next-devel/README.md)
+ - [ ] Approve and merge the fast-track [Pr(s)](https://github.com/coreos/fedora-coreos-config/pulls)
```

Comment:
Hmm, sorry I misunderstood what you were suggesting. I think I'd still prefer that a second person does the code review. But the releaser should be responsible for ensuring it eventually gets merged (e.g. getting reviews or rebasing if needed).